app/api/schema/events.py

```diff
@@ -118,6 +118,7 @@ def validate_timezone(self, data, original_data):
     refund_policy = fields.String(dump_only=True,
                                   default='All sales are final. No refunds shall be issued in any case.')
     is_stripe_linked = fields.Boolean(dump_only=True, allow_none=True, default=False)
+    is_featured = fields.Bool(default=True)
```

Comment:
Should be False by default. Do we want every event anyone creates to be featured?

Reply:
No, we don't I will fix that